specification/vmwarevirtustream/resource-manager/Microsoft.VMwareVirtustream/preview/2019-08-09-preview/examples/PrivateClouds_Get.json

```diff
@@ -0,0 +1,57 @@
+{
+  "parameters": {
+    "api-version": "2019-08-09-preview",
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "group1",
+    "privateCloudName": "cloud1"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/{subscription-id}/resourceGroups/group1/providers/Microsoft.VMwareVirtustream/privateClouds/cloud1",
+        "location": "eastus2",
+        "name": "cloud1",
+        "properties": {
+          "circuit": {
+            "authorizations": [],
+            "expressRouteID": "/subscriptions/{subscription-id}/resourceGroups/tnt13-41a90db2-9d5e-4bd5-a77a-5ce7b58213d6-eastus2/providers/Microsoft.Network/expressroutecircuits/tnt13-41a90db2-9d5e-4bd5-a77a-5ce7b58213d6-eastus2-xconnect",
+            "primarySubnet": "192.168.53.0/30",
+            "secondarySubnet": "192.168.53.4/30",
+            "expressRoutePrivatePeeringID": "/subscriptions/{subscription-id}/resourceGroups/tnt42-cust-p01-dmo01/providers/Microsoft.Network/expressroutecircuits/tnt42-cust-p01-dmo01-er/peerings/AzurePrivatePeering"
+          },
+          "cluster": {
```

Comment:
>cluster [](start = 11, length = 7)

Can this be called `managementCluster` to make it obvious what this is?

Also does the management cluster appear in the array of clusters?

Reply:
No.

Reply:
It can be changed to `maangementCluster` in the next API version.

No, the management cluster does not appear in the array of clusters.

Reply:
That will be a breaking change at that point. Why can't we change it now?

---
In reply to: [379495996](https://github.com/Azure/azure-rest-api-specs/pull/8296#discussion_r379495996) [](ancestors = 379495996)

Reply:
It is not just a breaking change in the client SDK, it would be a breaking change in the published API and all clients that currently use it. A change like this belongs in a new version. We could call that version 2020-03-20-preview. Let's put this change in 2020-03-20-preview.

Reply:
If we can lump it in with other already-planned breaking changes, then we could just get all over with in one shot. If the property rename is the only breaking change, then I'm not sure it's worth it. (It would be better if the name was clearer, but users will figure that out after this first time.)